docker-compose.test.yaml

```diff
@@ -5,6 +5,36 @@ version: '3.4'
 
 services:
   ffc-demo-user-service:
-    command: sh -c "scripts/wait-for/wait-for ffc-demo-user-postgres:5432 -- npm run migrate && npm run test"
+    command: sh -c "scripts/wait-for/wait-for ffc-demo-user-test-postgres:5432 -- npm run migrate && npm run test"
+    depends_on:
+      - ffc-demo-user-test-postgres
+    environment:
+      NODE_ENV: test
+      POSTGRES_DB: ffc_demo_users_test
+      POSTGRES_PASSWORD: ppp
+      POSTGRES_USERNAME: postgres
+      POSTGRES_HOST: ffc-demo-user-test-postgres
+      POSTGRES_PORT: 5432
     volumes:
       - ./test-output:/usr/src/app/test-output

```

Comment:
Do we need to add volumes here to bind-mount code from the host so we test current changes in development? Might require a `docker-compose.test.override.yaml` for dev so that tests can still run in the CI pipeline.

Reply:
Perhaps not, but could be something to leave until we're looking at the Jenkins pipelines as they will most likely require changes to this process anyway.

Reply:
I think my question came across wrong. I meant, we should add `node_modules` and `./` volumes here so that running the tests in dev will use local code changes. Otherwise, the tests will always run against the code built into the container image and ignore local changes.

Reply:
Ah yes.